reporting/console.go

```diff
@@ -24,27 +25,28 @@ func (c *ConsoleReporter) SendSummaryReport(
 	numRepos int,
 	report FindingSummary,
 	reportTime time.Time,
+	teamSummaries TeamSummaries,
 	wg *sync.WaitGroup,
 ) error {
 	defer wg.Done()
 	summaryReport := color.Bold.Sprint(header) + "\n"
-	summaryReport += color.Style{color.OpItalic}.Sprint(reportTime.Format(time.RFC1123)) + "\n\n"
+	summaryReport += color.Style{color.OpItalic}.Sprint(reportTime.Format(DATE_LAYOUT)) + "\n\n"
```

Comment:
[q] why the change away from a standard RFC format?
Do we just want to cut off all the time + tz info?

Reply:
yes, it was intentional haha, steve suggested that here https://getpinwheel.slack.com/archives/C01J0UC6MK8/p1708461855360679?thread_ts=1708375594.246659&cid=C01J0UC6MK8